demo/index.min.js

```diff
@@ -742,6 +752,7 @@ class AuroAccordion extends r {
           id="accordionTrigger"
           aria-controls="accordionContent"
           aria-expanded="${this.expanded}"
+          aria-disabled="${this.disabled ? 'true' : 'false'}"
```

Comment:
```suggestion
          ?aria-disabled="${this.disabled}"
```

You can just conditionally add the attribute based on `this.disabled`.

It's out of scope of this bug fix but `aria-expanded` should also actually be done the same way. Feel free to make that change too if you don't mind.